lib/osf-components/addon/components/file-browser/template.hbs

```diff
@@ -64,104 +64,22 @@
-                            <FaIcon @icon='ellipsis-v' />
-                        </Button>
-                    </dropdown.trigger>
-                    <dropdown.content local-class='DropdownList'>
-                        <Button
-                            local-class='FileOptionsDropdown__option'
-                            @layout='fake-link'
-                        >
-                            <FaIcon @icon='download' />
-                            {{t 'general.download'}}
-                        </Button>
-
-                        <ResponsiveDropdown
-                            local-class='FileOptionsDropdown__option'
-                            @horizontalPosition='right'
-                            as |dd|
-                        >
-                            <dd.trigger
-                                data-analytics-name='File sharing options'
-                            >
-                                <Button
-                                    local-class='FileOptionsDropdown__option'
-                                    @layout='fake-link'
-                                >
-                                    <FaIcon @icon='share-alt' />
-                                    {{t 'general.share'}}
-                                </Button>
-                            </dd.trigger>
-                            <dd.content local-class='DropdownList'>
-                                <Button
-                                    @layout='fake-link'
-                                >
-                                    <FaIcon @icon='envelope' />
-                                    {{t 'social.email'}}
-                                </Button>
-                                <Button
-                                    @layout='fake-link'
-                                >
-                                    <FaIcon @prefix='fab' @icon='twitter-square' />
-                                    {{t 'social.twitter'}}
-                                </Button>
-                                <Button
-                                    @layout='fake-link'
-                                >
-                                    <FaIcon @prefix='fab' @icon='facebook-square' />
-                                    {{t 'social.facebook'}}
-                                </Button>
-                                <Button
-                                    @layout='fake-link'
-                                >
-                                    <FaIcon @prefix='fab' @icon='linkedin' />
-                                    {{t 'social.linkedin'}}
-                                </Button>
-                            </dd.content>
-                        </ResponsiveDropdown>
-                    </dropdown.content>
-                </ResponsiveDropdown>
-            </div>
         </li>
+    {{/if}}
+    {{#each @manager.displayItems as |item|}}
+        <FileBrowser::Item @item={{item}} @manager={{@manager}} />
```

Comment:
I think the components would be a lot cleaner if we had a separate `FileBrowser::FolderItem` and `FileBrowser::FileItem`. Then you wouldn't have all of the if statements in the `Item` component, just one if statement here. It's not as DRY, but DRYness is sometimes overrated.

Reply:
I thought about that. Although we might DRY up the template a bit, we are adding two more `styles` file which shares a lot of the same styles.

Reply:
Yeah, we don't change styles a lot. I don't think that would be terrible.